config/routes.rb

```diff
@@ -4,8 +4,8 @@
   get 'todos/show/:id' => 'todos#show'
   post "todos" => "todos#create"
   put "todos/:id" => "todos#update"
-  delete '/todos/destroy_doneTask' => 'todos#destroy_Done_Task'
-  delete '/todos/destroy_all' => 'todos#destroy_All_Task'
+  delete '/todos/destroy_done_todo' => 'todos#destroy_done_todo'
```

Comment:
ここはそれぞれのプロジェクトによって考え方変わるのですが、以下のようにして良いと思います！（１考え方として覚えておいてほしいです）

（理由）
- HTTPのリクエスト（DELETE）によってアクションが分かるのでわざわざパスに書く必要がない
- `/todos`の時点でtodoに関する削除系のメソッド（DELETEだから）と分かるため無駄な情報はかかない
- `done`は必要な情報なので残す

（結論）
パス（＆HTTPリクエスト）から、「todoのdoneになっているものを削除するエンドポイント」ということが伝わることが大事で、余計な情報は必要に応じて省く
→　もちろんわざと冗長に書く場合もあるので臨機応変に

```suggestion
  delete '/todos/done' => 'todos#destroy_done_todo'
```

FYI：Railsのルーティングがある程度ルールが決まっているのでこちらは読んでおいてほしい & 実践できてほしいです！
https://railsguides.jp/routing.html